### PR TITLE
Use new Language stub for samples

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]

**Comment by heaths** [comment]
Blocked on https://github.com/Azure/azure-sdk-tools/pull/3702

File: sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ products:
 - azure
 - azure-cognitive-services
 - azure-text-analytics
+- language-service
```
Comment (maririos) [inline sdk/textanalytics/Azure.AI.TextAnalytics/samples/README.md:9]: what is the difference between this and `azure-language-service` ? I remember @deyaaeldeen added the later one and is the one used in JS https://github.com/deyaaeldeen/azure-sdk-for-js/blob/5e57c9010d8273db4e86805c14eea9bd98033115/sdk/textanalytics/ai-text-analytics/package.json#L52

**Comment by heaths** [comment]
The official stub does not have "azure-" as a prefix. I don't know why, but it's not listed in the page with all the stubs.

**Comment by heaths** [comment]
image

**Pull request merged**